components/blocks/cards/LogInCard/LogInCard.vue

```diff
@@ -45,7 +45,7 @@
           </div>
         </div>
 
-        <BaseButton class="login-button" @click="login">Log In</BaseButton>
+        <BaseButton class="login-button" @click="login"> Log In </BaseButton>
```

Comment:
is this postspace intentional? i find it weird since you could use a padding instead.

Reply:
Fairly certain that this spacing thingy is from `prettier`, cause I didn't intentionally do that. Same goes for a lot of text that got moved to a new line.

Reply:
ah, makes sense